### PR TITLE
uses expressions in UpdateDeParser, fix escaping in ExpressionDeParser, fix visualization-button-bug, fix datetime-replacing, fix column definition behaviour

### PR DESCRIPTION
**Pull request by binkertpat**

### CODE REVIEW

File: src/main/java/sqltoregex/deparser/CreateTableDeParserForRegEx.java [hunk 1]
```diff
@@ -271,9 +271,33 @@ private String deParseColumnDefinition(ColumnDefinition columnDefinition) {
...
+                Iterator<String> stringIterator = Arrays.stream(splittedColumnDefinition).iterator();
+                while(stringIterator.hasNext()){
+                    String splitElement = stringIterator.next();
+
+                    if(splitElement.contains("(") && splitElement.contains(")")){
+                        tempColumnDefinition.append("\\(");
+                        tempColumnDefinition.append(
+                                SpellingMistake.useOrDefault(
+                                        this.keywordSpellingMistake,
+                                        splitElement.replace("(", "").replace(")","")
```
Comment (Alletkla) [inline src/main/java/sqltoregex/deparser/CreateTableDeParserForRegEx.java:286]: In which cases an semicolon exists? 
Where is this added again in the following LoC

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -4,6 +4,7 @@ server.compression.enabled=true
 server.compression.mime-types=text/html,text/xml,text/plain,text/css,text/javascript,application/javascript,application/json
 server.compression.min-response-size=1024
 server.http2.enabled=true
+server.max-http-header-size=10MB
```
Comment (Alletkla) [inline src/main/resources/application.properties:7]: remove

**Review by Alletkla — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 2 above)
Comment (binkertpat) [reply src/main/resources/application.properties:7]: done

File: src/main/java/sqltoregex/deparser/CreateTableDeParserForRegEx.java (hunk 1 above)
Comment (binkertpat) [reply src/main/java/sqltoregex/deparser/CreateTableDeParserForRegEx.java:286]: i didnt see what u mean or want you wanna change.. there a no semicolons or commas :D

Comment (Alletkla) [reply src/main/java/sqltoregex/deparser/CreateTableDeParserForRegEx.java]: nevermind I'm blind light was to bright on the laptop and I confused the parameter comma ._.

**Review by Alletkla — APPROVED** [review]

**Pull request merged**